oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/OALRuntime.java

```diff
@@ -421,13 +421,13 @@ private Class generateDispatcherClass(String scopeName,
                         )
                     }
                 );
-
             dispatcherClass.setGenericSignature(dispatcherSignature.encode());
         } catch (NotFoundException e) {
             log.error("Can't find Dispatcher interface for " + className + ".", e);
             throw new OALCompileException(e.getMessage(), e);
         }
 
+
```

Comment:
This should be reverted.

Reply:
Sorry , This PR  just for preview or  discussion  . You could ignore it